src/main/java/subway/domain/Line.java

```diff
@@ -34,14 +35,14 @@ protected Line() {
     public Line(
             final String name,
             final String color,
-            final List<Station> stations,
-            final Long upStationId,
-            final Long downStationId,
+            final Station upStation,
+            final Station downStation,
             final int distance
     ) {
         this.name = name;
         this.color = color;
-        this.stations = new Stations(stations, upStationId, downStationId);
+        this.sections = new Sections();
+        sections.add(new Section(distance, upStation, downStation, this));
```

Comment:
비즈니스 상 Sections 가 생성될 떄는 항상 구간이 하나는 있어야 할 것 같은데요,
다음과 같이 해주면 어떨까요? 😄 
```suggestion
        this.sections = new Stations(stations, upStationId, downStationId);
```